bundles/org.eclipse.orion.client.ui/web/orion/status.js

```diff
@@ -92,6 +92,29 @@ define([
 			}
 			return this._notifierElements;
 		},
+
+		/**
+		 * Returns a message that is as detailed as possible.
+		 * @param {String|orionError} either a string or a JSON representation of an IStatus

```

Comment:
include name of the variable, e.g.

@param {String|orionError} status Either a string or a JSON representation of an IStatus